### DISCUSSION TITLE
Sudden termination occurred while executing examples - complicated.py in 32-bit python

### DISCUSSION THREAD

**Discussion by youniceknight** [highfestiva/finplot#413] [Q&A]
First of all, thank you very much for making such a great charting library.

python 3.8 (32-bit)
Pycharm community edition 2021.1
os: windows 10 home

When executing examples - complicated.py, (After running for about 10 minutes) the following error occurs.
It didn't happen in python 3.8 (64-bit), but it happens in (32-bit) environment.

Process finished with exit code -1073741819 (0xC0000005)

(line 111) self.df = pd.concat([df, candle])

After changing the code in the above line to the following code, it was confirmed that the above sudden termination was slightly reduced, but it does not seem to be a complete solution.

  self.df = pd.concat([df.loc[:], candle])

I don't know why.

**Comment by highfestiva** [comment]
Sorry to hear that, seems Pandas has some access violation in their 32-bit code. Unfortunately not much I can do. If you feel like it, make a loop that just does that and crashes in a second, and post it as a bug report to Pandas. Thanks!

**Reply by youniceknight** [reply]
Thank you so much for your answer. May you always be full of blessings.
